benchmarks/models.py

```diff
@@ -203,6 +203,21 @@ def testjobs_updated(self):
         jobs = self.test_jobs.filter(updated_at__gt=self.updated_at)
         return jobs.exists()
 
+    @property
+    def testrunnerclass(self):
+        runners = sorted(
+            filter(
+                lambda t: t not in ['GenericLavaTestSystem', 'TestSystem'],
+                set([ t.testrunnerclass for t in self.test_jobs.all() ])
+            )
+        )
+        if len(runners) == 1:
+            return runners[0]
+        elif len(runners) > 1:
+            raise LookupError("Result has multiple testrunner classes: %r'" % runners)
```

Comment:
Yes, we just need to compare apples to apples. So in case both builds contain TestJobs of type 'microbenchmarks' with identical metadata (32 vs 64 bit) than we compare them accordingly. We don't need the ResultData objects for doing the comparison - only TestJob. I agree this might cause a confusion which data to compare using which method. Are we sending the 'comparison email' on each TestJob completion or when all are completed? I guess we should turn into the former and than the comparison would become easier (I think).